rfc/training-content-delivery.md

```diff
@@ -0,0 +1,75 @@
+
+### Delivery
+
+To deliver the material, each element has its own needs:
+
+- Presentations require there being only a single bit of text presented at a time. To achieve this, I recommend using Google Slides.
+- Labs require much more text, as well as other hypertext tools that we've already developed on `/docs`. To achieve this, I recommend using Markdown rendered into HTML through Jekyll.
+- Demos each have different needs, but are often broadcast in the classroom from the instructor's computer. I recommend either recording demos that can be shown on YouTube, or solving demos on a case-by-case basis.
+
+### Structure
+
+As a wrapper around all of this content, the training material is hosted on `cockroachlabs.com/training`. We currently do not have password-restricted access on this content, but may in the future.
+
+This site is written in Markdown and converted to HTML through Jekyll and has the following structure:
+
+`home/<day>/<module>/`
+
+That is:
+- 1 home page for all training material (if this training system developers further, it's possible to insert levels above this)
+- 1 page for each day, which contain links to its modules
+- 1 page page for each module, which contains contains a brief summary, as well as links to both the presentations and the labs.
+
+Off of the `../<module>/` subdirectory, there would also be a page for any labs from that day. We would not have a page for presentations because these could just be links to Google sheets.
+
+### UX
+
+Users go to the training site and the instructor guides them through each of the presentations. After the presentation, they return to the module page and follow through any labs related to the module.
+
+At the end of each module page and each lab, there are links to the subsequent modules.
+
+## Drawbacks
+
+This implementation _does not_ provide password protection, and therefor relies on obscurity to "protect" the content's IP. Because the training content is potentially monetizable, this could jeopardize that strategy.
+
+## Rationale and Alternatives
+
+This design creates a straighforward user experience that's consistent with our docs site. It leverages our existing toolchain and requires little-to-no ramp up effort.
+
+However, to overcome the drawbacks (i.e., as a means of providing password authentication), training content could be delivered solely through Google Drive, swapping out HTML-rendered Markdown for Google Docs, and using Google Sheets as navigation elements. However, I think that the UX of this is very poor and places a burden onto the instructor and the attendees because of our hopes of monetizing the content.
+
+## Unresolved questions
+
+- Which repo should this content belong to?
+- Should this content be versioned?
```

Comment:
It wouldn't hurt to version this content, given that some guidance may change from version to version. That may be one reason to put it in the docs repo, where we already have versioning logic in place.